### PR TITLE
#165305752-Fix swagger description

### PR DESCRIPTION
**Pull request by Amoswachira**
#### Description
Add a description to the swagger endpoints

#### Type of change
- [x] Bug fix (nonbreaking change which fixes an issue)
- [ ] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ]  Integration test
- [x]  Unit test

#### Checklist:
- [x] My code follows the style guide for this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[#165305752](https://www.pivotaltracker.com/n/projects/2326460/stories/165305752)

#### Screenshots
screencapture-127-0-0-1-8000-api-swagger-2019-05-06-00_26_05

### CODE REVIEW

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

File: authors/apps/profiles/tests/test_follow.py [hunk 1]
```diff
@@ -145,3 +145,59 @@ def test_get_all_followers(self):
 
         res = self.client.get('/api/profiles/ironman/followers/')
         self.assertEqual(res.status_code, status.HTTP_200_OK)
+
+    def test_user_not_found_follow(self):
```
Comment (Ogutu-Brian) [inline authors/apps/profiles/tests/test_follow.py:149]: Hello @Amoswachira ,
These tests are not part of the scope of this pull request, so I don't think it should be included

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

**Review by Bakley — CHANGES_REQUESTED** [review]
Hey @Amoswachira Good work on updating the swagger documentation but there some inconsistency in them 
```
"""Login a User"""
```
Most of your changes are that way and some of the docstrings is:
```
    """
    Confirms password reset after retrieving token embedded in mail
    """
```

File: authors/apps/profiles/tests/test_follow.py (hunk 1 above)
Comment (Amoswachira) [reply authors/apps/profiles/tests/test_follow.py:149]: @Ogutu-Brian the reason those are there is that I forgot to checkout from develop branch and  i checkout from my fix profiles branch

**Comment by Amoswachira** [comment]
@Bakley Thanks for the feedback let me implement that

**Review by Ogutu-Brian — APPROVED** [review]

**Comment by Amoswachira** [comment]
@Bakley i have fixed the changes

**Review by Bakley — APPROVED** [review]

File: authors/apps/profiles/tests/test_follow.py (hunk 1 above)
Comment (oluwagbenga-joloko) [reply authors/apps/profiles/tests/test_follow.py:149]: Can you update this pull request and include only the changes related to your task.  @Amoswachira

Comment (Amoswachira) [reply authors/apps/profiles/tests/test_follow.py:149]: @oluwagbenga-joloko okay let me do that

**Pull request closed without merging**